src/OpenLoco/Map/TileManager.cpp

```diff
@@ -91,12 +92,45 @@ namespace OpenLoco::Map::TileManager
         TileManager::updateTilePointers();
     }
 
+    // Note: Must be past the last tile flag
+    static void markElementAsFree(TileElement& element)
+    {
+        element.setBaseZ(255);
+        if (element.next() == *_elementsEnd)
+        {
+            _elementsEnd--;
+        }
+    }
+
     // 0x00461760
     void removeElement(TileElement& element)
     {
-        registers regs;
-        regs.esi = X86Pointer(&element);
-        call(0x00461760, regs);
+        if (&element == *_F00158)
```

Comment:
any ideas what `F00158` is? At face value its a TileElement pointer, but what does it represent/how is it used?

Reply:
I had a look and couldn't work out what it represented for now. Tbh i think it might be dead code